molecule.yml

```diff
@@ -0,0 +1,22 @@
+---
+ansible:
+  config_file: ansible.cfg
```

Comment:
Addressed in [`de57c6c`](https://github.com/azavea/ansible-scala/pull/3/commits/de57c6c56ef1134862d8bb76edcb2fbcfc5a15b7)